### PR TITLE
Task #223615 feat: Integrated show district api and no found data added in ka-table, added in utils

### PR DESCRIPTION
**Pull request by rahulg1254**

### CODE REVIEW

File: src/pages/district.tsx [hunk 1]
```diff
@@ -182,16 +197,17 @@ const District: React.FC = () => {
...
     selectedState,
-    selectedDistrict,
     showStateDropdown: false,
     selectedFilter,
     handleFilterChange,
-    handleSearch,
+    handleSearch: () => {},
   };
 
+  const showPagination = sortedDistricts.length > 10;
```
Comment (itsvick) [inline src/pages/district.tsx:209]: @rahulg1254 add 10 in app.constant

File: src/utils/formatLabel.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const formatLabel = (label: string) => {
+  return label
```
Comment (itsvick) [inline src/utils/formatLabel.ts:4]: move this to helper.ts @rahulg1254

File: src/components/KaTableComponent.tsx [hunk 3]
```diff
@@ -110,6 +110,9 @@ const KaTableComponent: React.FC<KaTableComponentProps> = ({
               },
             },
           }}
+          noData={{
+            text: "No Data Found",
```
[the remark this answers is not in the document]
Comment (rahulg1254) [reply src/components/KaTableComponent.tsx:125]: Added

File: src/utils/formatLabel.ts (hunk 2 above)
Comment (rahulg1254) [reply src/utils/formatLabel.ts:4]: Moved

File: src/pages/district.tsx (hunk 1 above)
Comment (rahulg1254) [reply src/pages/district.tsx:209]: resolved

**Pull request merged**